appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -17,16 +18,18 @@ else{
 }
 
 function Rebase-Uri($originalURI){
-  $replace = $originalURI.IndexOf("file:///")
-  if($replace -ne  -1){
-      $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString("file:///".Length))
-      $replaceLength = $caseSensitiveUri.IndexOf($projectSlug)
-      if($replaceLength -ne -1){
-          $builder.Path = ($repoName, $repoCommit, $caseSensitiveUri.SubString($replaceLength+$projectSlug.Length+1) -join '/')
-          return $builder.ToString()
-      }
+  if($originalURI){
+    $replace = $originalURI.IndexOf("file:///")
+    if($replace -ne  -1){
+        $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString("file:///".Length))
+        $replaceLength = $caseSensitiveUri.IndexOf($projectSlug)
```

Comment:
`IndexOf` doesn't return a length, it returns a 0-based index. A nice descriptive name for this variable would be `$projectSlugIndex`.